docs/en/deploy/Running-Alluxio-On-Kubernetes.md

```diff
@@ -1504,12 +1504,29 @@ Here are some common properties that you can customize:
   </tr>
 </table>
 
-Then please use `helm-generate.sh` (see [here](https://github.com/Alluxio/alluxio/tree/master/integration/kubernetes#generate-kubectl-yaml-templates-from-helm-chart) for usage)
-to generate related templates. All CSI related templates will be under `${ALLUXIO_HOME}/integration/kubernetes/csi`.
-
 **Step 2: Deploy CSI services**
+You can use Helm to start the Alluxio CSI components with Alluxio cluster, or `kubectl` to create the resources manually,
+or parts from Helm and parts manually.
+
+{% navtabs deployCSI %}
+{% navtab helm %}
+
+To start the CSI components via helm chart, set the following values in your helm configuration file:
```

Comment:
any specific helm configuration file example in our doc?

Reply:
Our helm chart provides a set of default configurations